src/GeekLearning.Email/IEmailSender.cs

```diff
@@ -4,12 +4,12 @@
 
     public interface IEmailSender
```

Comment:
there should be additional overloads using `AddressTarget`

Reply:
That's a thought. But there needs to be a solid way to link an AddressTarget to an email address. And how does it look if we have multiple email addresses in the call?